oem.json

```diff
@@ -3074,38 +3153,102 @@
+            "https://beta.pithus.org/report/842d17e1944d748a7813ee8deb072224fd0665cf6c0504a0d90e46568d4c444d"
+        ],
+        "removal": "replace",
+        "suggestions": "app_stores"
     },
     {
         "id": "com.motorola.bug2go",
-        "description": "Bugs reporting app that sends info about crash report.\n",
+        "label": "com.motorola.bug2go",
+        "description": "Bugs reporting app that sends info about various crash reports and logs.",
         "removal": "delete"
     },
     {
         "id": "com.motorola.callredirectionservice",
-        "description": "Added in Android 10. Provide support for call redirection/cancellation if your Carrier supports it (when someone call.\nSee https://motorola-global-portal.custhelp.com/app/answers/prod_answer_detail/a_id/140542\nhttps://en.wikipedia.org/wiki/Call_forwarding\n",
+        "label": "com.motorola.callredirectionservice",
+        "description": "Added in Android 10. Provide support for call redirection/cancellation if your Carrier supports it.",
+        "web": [
+            "https://motorola-global-portal.custhelp.com/app/answers/prod_answer_detail/a_id/140542",
+            "https://en.wikipedia.org/wiki/Call_forwarding"
+        ],
         "removal": "delete"
     },
     {
         "id": "com.motorola.camera2",
-        "description": "Moto Camera 2 (https://play.google.com/store/apps/details?id=com.motorola.camera)\n",
-        "removal": "replace"
+        "label": "Moto Camera 2",
+        "description": "Moto camera 2 is optimized for Moto devices and integrated with Google Photos.",
+        "web": [
+            "https://play.google.com/store/apps/details?id=com.motorola.camera2"
+        ],
+        "removal": "replace",
+        "suggestions": "cameras"
+    },
+    {
+        "id": "com.motorola.camera3",
+        "label": "Moto Camera 3",
+        "description": "Moto redesigned camera only compatible with select devices launched in 2020 and beyond.",
+        "web": [
+            "https://play.google.com/store/apps/details?id=com.motorola.camera3"
+        ],
+        "removal": "replace",
+        "suggestions": "cameras"
+    },
+    {
+        "id": "com.motorola.camera3.content.ai",
+        "label": "Moto Camera Content 3",
+        "dependencies": [
+            "com.motorola.camera3"
+        ],
+        "description": "Provides AI related features to 'com.motorola.camera3'",
+        "web": [
+            "https://play.google.com/store/apps/details?id=com.motorola.camera3.content.ai",
+            "https://beta.pithus.org/report/0b0a4e3d19819faf9f7448817b88172b33d83973334b2ba5fd99a814aae5575b"
+        ],
+        "removal": "delete"
+    },
+    {
+        "id": "com.motorola.carrierconfig",
+        "label": "Carrier Services",
+        "description": "Related to various communication related actions.",
+        "web": [
+            "https://source.android.com/docs/core/connect/carrier"
+        ],
+        "removal": "caution"
```

Comment:
`caution` set without `warning`.

Reply:
As per the [instructions](https://github.com/MuntashirAkon/android-debloat-list#values-for-removal) `warning` **must be set** for `unsafe` packages.
But I add warnings for others as well when necessary.

In this particular case I didn't experience any network related issues but `caution` added because the user might experience other issues depending on various other factors. I haven't tested _Roaming_ because I'm not traveling, not tested on _non VoLTE_ SIM, other _network providers_, _voice mails_⸮ etc...

Reply:
> I haven't tested _Roaming_ because I'm not traveling, not tested on _non VoLTE_ SIM, other _network providers_, _voice mails_⸮ etc...

You can add this in the warning section.

Reply:
> "warning": "You might experience various network related issues, remember to enable this if faced any issue."

I'll remove this package from this list & add it to carrier.json instead.